sources/logdog/src/log_request.rs

```diff
@@ -286,6 +286,8 @@ where
 
 /// Uses the reqwest library to send a GET request to `URL` and returns the response.
 fn send_get_request(url: &str) -> Result<Response> {
+    let _ = rustls::crypto::aws_lc_rs::default_provider().install_default();
```

Comment:
do we want to `install_default()` in respective `main` / `run` functions for our sources ? 

https://docs.rs/rustls/latest/rustls/crypto/struct.CryptoProvider.html#using-the-per-process-default-cryptoprovider
> applications should call [CryptoProvider::install_default()](https://docs.rs/rustls/latest/rustls/crypto/struct.CryptoProvider.html#method.install_default) early in their fn main(). If applications uses a custom provider based on the one built-in, they can activate the custom-provider feature to ensure its usage.

I see that's done for other cases of `install_default()` in this PR

Reply:
I didn't want to install the default provider unless really necessary. For `logdog`, that's only when the request type is `http` or `https`.